docs/getting-started-ei.md

```diff
@@ -0,0 +1,97 @@
+# Getting Started - WSO2 Enterprise Integrator
+
+Setting up a basic pipeline for WSO2 Enterprise Integrator on Kubernetes is quick and simple.
+
+You can set up a simple CI/CD pipeline for WSO2 Enterprise Integrator in two steps.
+- Create a Docker image for WSO2 Enterprise Integrator.
+- Deploy the CI/CD pipeline.
+
+Before you begin to develop your pipeline, set up the following prerequisites in your Kubernetes cluster.
+
+## Prerequisites
+
+- Install and set up Helm.
+- Install Nginx Ingress Controller Git release nginx-0.22.0.
+
+
+## Create WSO2 Enterprise Integrator Docker Image
+
+The first step in developing the pipeline is to create an image for WSO2 Enterprise Integrator on a private Docker registry. This pipeline utilizes Docker Hub as its private registry.
+
+After creating the repository, pull the latest image from WSO2 and push it to the private registry.
+
+- If you have a WSO2 subscription, log into the WSO2 Docker registry using your subscription credentials.
+
+    ```
+    docker login docker.wso2.com
+    docker pull docker.wso2.com/wso2ei-integrator:6.5.0
+    docker tag docker.wso2.com/wso2ei-integrator:6.5.0 <DOCKER_ORGANIZATION>/wso2ei-integrator
+    ```
+
+    ><DOCKER_ORGANIZATION> refers to the name of your organization in Docker Hub.
+
+- If you do not have a WSO2 subscription use the docker hub image wso2/wso2ei-integrator:6.5.0 instead.
```

Comment:
@Aaquiff should be `Docker Hub`.

Plus, shall we link to the WSO2 Subscription page from here [1]. WDYT?

This applies to documentation of all products.

[1]: [Official documentation](https://wso2.com/subscription)

Reply:
Added this to the note that's at the end of this step